readthedocs/core/management/commands/update_repos.py

```diff
@@ -57,7 +57,8 @@ def handle(self, *args, **options):
         record = options['record']
         force = options['force']
         version = options['version']
-        if args:
+
+        if options:
```

Comment:
Options already have some keys (`record`, `force`, etc). I think we want to skip this and just do `for slug in options.get('slugs', []): ...`

Reply:
Thanks! Fixed.